backend/src/graphql/resolver/AdminResolver.ts

```diff
@@ -41,13 +232,39 @@ export class AdminResolver {
   @Mutation(() => [Number])
   async createPendingCreation(
     @Args() { email, amount, memo, creationDate, moderator }: CreatePendingCreationArgs,
+    @Ctx() context: any,
   ): Promise<number[]> {
     const userRepository = getCustomRepository(UserRepository)
     const user = await userRepository.findByEmail(email)
 
     const creations = await getUserCreations(user.id)
     const creationDateObj = new Date(creationDate)
     if (isCreationValid(creations, amount, creationDateObj)) {
+      // PROTO START
+      const loginUserRepository = getCustomRepository(LoginUserRepository)
+      const loginUser = await loginUserRepository.findOneOrFail({ email })
+      const userRepository = getCustomRepository(UserRepository)
+      const adminUser = await userRepository.findByPubkeyHex(context.pubKey)
+      const loginAdminUser = await loginUserRepository.findByEmail(adminUser.email)
+      const loginPendingTaskRepository = getCustomRepository(LoginPendingTaskRepository)
+      const loginPendingTask = loginPendingTaskRepository.create()
+      loginPendingTask.userId = user.id
+      loginPendingTask.request = pendingTasksRequestProto(
+        amount * 10000,
+        memo,
+        creationDateObj,
+        loginUser.pubKey,
+        loginAdminUser.pubKey,
+        loginAdminUser.privKey,
+      )
+      loginPendingTask.created = creationDateObj
+      loginPendingTask.resultJson = ''
+      loginPendingTask.paramJson = '{"blockchain_type":1}'
+      loginPendingTask.taskTypeId = 10
+
+      await loginPendingTaskRepository.save(loginPendingTask)
```

Comment:
This works and creates the entry in the login_pending_task tabelle!
But we can not confirm this creation on the old backend... http://localhost/account/checkTransactions

Maybe we are still missing something